lib/jasmine-reportportal-reporter.js

```diff
@@ -71,44 +67,40 @@ class ReportportalReporter {
     }
 
     getSuiteAttributesBySuite (suite) {
-        const { suiteAttributes } = this.suiteAdditionalCustomParams;
-
-        return suiteAttributes && suiteAttributes.filter(attr => attr.suite === suite).map(attr => ({
+        return this.suiteAttributes && this.suiteAttributes.filter(attr => attr.suite === suite).map(attr => ({
```

Comment:
Why not to use Map for suites attributes?

Reply:
In my case, the using map for attributes is not as clear as for description. Implementation gives rise to many necessary checks and the code becomes dirty.